net/xia/ppal_ether/main.c

```diff
@@ -226,6 +227,36 @@ static void detach_neigh_to_interface(struct fib_xid_ether_main *mether)
 	ether_interface_put(einterface);
 }
 
+/* Creates a cached hdr and alocates to a mfxid. */
+static void xia_ether_header_cache(struct fib_xid_ether_main *mfxid)
```

Comment:
Update the above comment with this improved comment: `/* Initialize the cached Ethernet header. */`